src/main/resources/public/css/pages/app-install.css

```diff
@@ -38,26 +52,55 @@
 }
 
 .app-install-wrapper .app-install-description p {
-    width: 70%;
+    width: 80%;
     line-height: 1.8em;
-    font-size: 1.25em;
+    font-size: 1em;
 }
 
-.app-install-wrapper .app-install-description div.long{
-    justify-content: center;
+.app-install-wrapper .app-install-description div.long {
+    justify-content: space-evenly;
     display: flex;
+    flex-wrap: wrap;
 }
 
-.app-install-wrapper .app-install-description div.long > *{
+/* .app-install-wrapper .app-install-description div.long > *{
     text-align: justify !important;
-}
+} */
 
-.app-install-wrapper .go-to-long-description{
+table {
+    border-collapse: collapse !important;
+    margin: 25px 0 !important;
+    font-size: 0.9em !important;
+    font-family: sans-serif !important;
+    min-width: 400px !important;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15) !important;
+}
+thead tr {
+    background-color: #f4a3bd;
+    color: #ffffff;
```

Comment:
Variable :)

Reply:
Done for the white color, not the background-color as used only once in the project.